.github/workflows/ci.yaml

```diff
@@ -40,8 +40,10 @@ jobs:
         uses: actions/setup-node@v3
       - name: 🔎 Install dependencies
         run: npm ci
-      - name: Verify updated README
-        run: npx embedme README.md --verify
+      - name: Generate README
+        run: npm run readme:generate
+      - name: Check README up to date
+        run: npm run readme:check
```

Comment:
The script already throws an exit code when the readme gets updated (as part of the pre-pull hook), but I'm realizing that it's just doing a comparison between whats generated and what exists.... which is the same as a diff. So maybe I remove that from the script?

Reply:
It's maybe duplicated but I think it's good, better to be safe than sorry.